quality_of_service_demo/rclpy/quality_of_service_demo_py/lifespan.py

```diff
@@ -51,7 +52,10 @@ def main(args=None):
 
     qos_profile = QoSProfile(
         depth=parsed_args.history,
```

Comment:
Unrelated to this PR: I would recommend making arguments match the QoS setting for clarity. Briefly, this looked like a bug, assigning a history setting to a depth setting.